helfomat-docker/docker-compose.yml

```diff
@@ -32,35 +32,14 @@ services:
     - 9200:9200
     - 9300:9300
 
-  zoo1:
-    image: zookeeper:3.4.9
-    container_name: helfomat-zookeeper1
-    hostname: zoo1
-    ports:
-    - "2181:2181"
+  postgres:
+    image: postgres:10
```

Comment:
Why such an old postgres? you can use 12. And when you use 12-alpine, the image is even smaller.

Reply:
fixed